packages/cli/src/commands/index.ts

```diff
@@ -25,8 +25,8 @@ export default new Map([
   ['logout', 'logout'],
   ['logs', 'logs'],
   ['ls', 'list'],
-  ['project', 'projects'],
-  ['projects', 'projects'],
+  ['project', 'project'],
+  ['projects', 'project'],
```

Comment:
Does this affect the end user? I think the answer is "no" but just wanted to check.

Reply:
No, the end user can still use `vc project` and `vc projects` interchangeably. The only thing that has changed here is that `vc project` is the "real" command.

Reply:
I have been thinking that these plural commands should all be changed to singular. Then the singular should print help instead of assuming list.

So something like:

- `vc project` is the same as `vc project --help`
- `vc projects` is the same as `vc project ls`
- `vc project ls` list the projects
- `vc project connect` connect a project